magento/magento2
# Magento#24736: Fixed of `quote`.`customer_is_guest` set to 0 for guests

### Description (*)
Quote `customer_is_guest` set to 0 for guests

### Fixed Issues (if relevant)
1. magento/magento2#24736: Quote `customer_is_guest` set to 0 for guests

### Manual testing scenarios (*)
1. Testing as a guest:
 - Be guest
 - Put an item in cart
 - Check quote.customer_is_guest in database. The value should be 1;
2. Testing as a logged-in customer:
 - Sign in or Register
 - Put an item in cart
 - Check quote.customer_is_guest in database. The value should be 0

### Questions or comments

### Contribution checklist (*)
